docker-compose.yml

```diff
@@ -0,0 +1,30 @@
+version: "3.7"
+
+services:
+  meilisearch:
+    image: getmeili/meilisearch:${MEILISEARCH_VERSION}
+    container_name: meilisearch
+    ports:
+      - "7700:7700"
```

Comment:
Have you tried this: 
```yaml
ports:
  - "7701:7700"
  - "7700:7700"
```

Because I think we can have only one service answering in different ports... Or you did this to make possible to work in parallel (to avoid index names conflicts?)